fw/hashseg.py

```diff
@@ -55,7 +55,7 @@
 
 # According to the v2.0 PDF the metadata is 128 bytes long, but this does not
 # seem to work. All official firmware seems to use 120 bytes instead.
-METADATA_SIZE = 120
+MBM_V6_METADATA_SIZE = 120
```

Comment:
```suggestion
MBN_V6_METADATA_SIZE = 120
```

make sure -v6 still works!

Reply:
Below you reference MB**N**_V6_METADATA_SIZE, but here you call it MB**M**_V6_METADATA_SIZE, I don't think this will work :sweat_smile: